code/stable/dblpend/TraceyGraph/allvsall.dot

```diff
@@ -1,17 +1,17 @@
 digraph allvsall {
-	theory:v_x1 -> velocity;
-	theory:v_x1 -> p_x1;
-	theory:v_y1 -> velocity;
-	theory:v_y1 -> p_y1;
-	theory:v_x2 -> velocity;
-	theory:v_x2 -> p_x2;
-	theory:v_y2 -> velocity;
-	theory:v_y2 -> p_y2;
+	v_x1:theory -> velocity;
```

Comment:
in this case, `theory:v_x1` and `v_x1:theory` both make little sense... curious where it comes from. The reasoning above is sound, so it's super weird that the `theory` now comes at the end!

Reply:
That UID is for:
https://github.com/JacquesCarette/Drasil/blob/1330bc431605eb31bb8d7d9bca90da02cb5b9696/code/drasil-example/dblpend/lib/Drasil/DblPend/GenDefs.hs#L33-L34

Relevant 'backend' code:
https://github.com/JacquesCarette/Drasil/blob/1330bc431605eb31bb8d7d9bca90da02cb5b9696/code/drasil-theory/lib/Theory/Drasil/ModelKinds.hs#L89-L91

https://github.com/JacquesCarette/Drasil/blob/1330bc431605eb31bb8d7d9bca90da02cb5b9696/code/drasil-theory/lib/Theory/Drasil/ModelKinds.hs#L58-L59

Reply:
i.e., `theory` is a namespace

Reply:
So you want namespaces to appear on the right?

Reply:
If we have `nest C $ nest B $ nest A X`, the UID before this PR ends up as: `C:B:A:X`. I read the relationships between `C`, `B`, and `A` as $C \subset B \subset A$. The code would then be read as "nest $X$ in $A$, then create a new namespace for $X$ within $A$, etc.".  In which case, the update would make more sense because it shows $X$ near the smallest namespace, `C` (e.g., `X:C:B:A`).

Thinking about it again... maybe I was confused? The documentation is unclear about the relationship between the nested namespaces. I'm not sure, and I'm impartial because I'm fairly convinced we're going to have to completely rework the `UID` system once we get to the real work on editing the chunk graph, theory refinements, etc.

I'll undo it for now.

Reply:
Or did you mean that you agreed with my view and just wanted the larger namespaces to appear leftmost rather than rightmost? e.g., `A:B:C:X`.

Reply:
I want our name-spacing conventions to be the same as Haskell's and Agda's, so `Data.List.Membership` for example. I don't see this as having anything to do with subsets, but rather is all about hierarchical classification.

Reply:
Yeah, that does make more sense. I just updated that now in 361b507d0d72b7749ad99bd22b11d461e530ddfd and e2f687cda3ae5c9e1ebe01b26308a25ed7d36117 .